symplyphysics/laws/optics/interference_minimum.py

```diff
@@ -0,0 +1,38 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression,
+                           validate_input, validate_output, dimensionless)
+
+# Description
+## If the travel difference is equal to an odd number of half-waves,
+## then at this point of the screen there will be a minimum of intensity during interference.
+
+# Law: delta = (2m+1) * lambda / 2
+# Where:
+## delta - optical travel difference for two coherent waves
+## lambda - length of waves (wave 1 and wave 2)
+## m - number of interference minimum
```

Comment:
Mention somewhere that this number can be chosen arbitrarily to achieve the desired travel difference.

Reply:
As far as I know $m=0; 1; 2; ...; \infty$

Reply:
That's what I've send. You chose any positive integer number to pass into the law.